code/game/objects/items/airbubble.dm

```diff
@@ -389,6 +397,33 @@
 		)
 		new/obj/item/weapon/handcuffs/cable(src.loc)
 		update_icon()
+	else if(istype(W, /obj/item/device/janitool))
+		var/obj/item/device/janitool/T = W
+		if(!T.fold)
+			to_chat(user, "<span class='warning'>[T] has to be in fold mode.</span>")
+			return
+		if(zipped)
+			to_chat(user, "<span class='warning'>[src] Has to be unzipped.</span>")
+			return
+		if(zipped)
+			to_chat(user, "<span class='warning'>[src] Has a hole in it, there is no use.</span>")
+			return
+		user.visible_message("", "[W] sucks [src] into itself, spitting out entirely folded air bubble.")
```

Comment:
wat. Why would you call this function with the first argument as an empty string? That's equivalent to doing `to_chat(user, "[W] sucks [src] into itself, spitting out entirely folded air bubble.")`. Because, the second argument is given to the source, the first is broadcast to everyone.